train.py

```diff
@@ -62,11 +69,15 @@ def generate_subspectrogram(duration_s=config.SUBSPECTROGRAM_DURATION_S, fft_rat
         checkpoint.epoch.assign_add(1)
         checkpoint_manager.save()
 
+    print("======================== evaluation on validation data =========================")
+    # test model on validation set
+    utils.test_model(model, val_spectrograms, val_labels, val_loss, val_accuracy)
```

Comment:
Testing the model against validation periodically through the training could be beneficial in order to find the correct number of epochs for early stopping in one run

Reply:
Yes that will be in the next pr

Reply:
But without early stopping implemented, just testing the model to see the output slows the code down a lot.